keystrokes/src/components/Home/Home.js

```diff
@@ -27,6 +27,15 @@ function Home(props) {
 	}
 
 	useEffect(() => {
+		try {
+			ref.child(userPath).on('value', change => {
```

Comment:
I think these return the unsubscribe function that you'll want to return from the use effect

Reply:
so, this might be a place for an odd problem. I want this to continue to work while outside of Home.js, i want it working everywhere. I know that its bad if it creates multiple of these, but it doesnt appear to do that. im pretty sure if i have a return on this, it wont work on all of the different components. Ill double check that though.

Reply:
doesnt seem to cause any problems when returning the listener return object, still works from other places. Seems odd to me though, i would expect it to be removed when visiting other pages, seeing as im going from home to character management.